### PR TITLE
Faq

### PR DESCRIPTION
**Pull request by jospoortvliet**
Word changes here and there.

### CODE REVIEW

**Comment by jospoortvliet** [comment]
@MariusBluem @NinaNC if you could have a look and spot any big issues... would rock. If OK, update branch & merge pls ;-)

**Comment by MariusBluem** [comment]
What has this to do with „FAQ“ 🤔

File: page-compare.php [hunk 1]
```diff
@@ -174,7 +174,7 @@
 							<li title="" rel="tooltip"><i class="fa fa-apple" aria-hidden="true"></i> <i class="fa fa-windows" aria-hidden="true"></i> <i class="fa fa-linux" aria-hidden="true"></i></li> <!--Desktop clients-->
 							<li title="" rel="tooltip"><i class="fa fa-check" aria-hidden="true"></i></li> <!-- Sync any local folder-->
 							<li title="" rel="tooltip"><i class="fa fa-times" aria-hidden="true"></i></li> <!--LAN Synchronization-->
-							<li title="" rel="tooltip"><i class="fa fa-check" aria-hidden="true"></i></li> <!--Extensible with apps-->
+							<li title="Almost 40 available for ownCloud 10 in beta app store" rel="tooltip"><i class="fa fa-clock-o" aria-hidden="true"></i></li> <!--Extensible with apps-->
```
Comment (MariusBluem) [inline page-compare.php]: Why the „clock“ and not the ✅

File: page-compare.php [hunk 2]
```diff
@@ -185,7 +185,7 @@
 							<li title="optional" rel="tooltip"><i class="fa fa-check optional" aria-hidden="true"></i>/<i class="fa fa-check optional" aria-hidden="true"></i>/<i class="fa fa-times" aria-hidden="true"></i></li> <!--Calendar/Contact/Mail integration-->
 							<li title="Optional. Available for iOS, Android possible with not related third party tool" rel="tooltip"><i class="fa fa-check optional" aria-hidden="true"></i>/<i class="fa fa-check optional" aria-hidden="true"></i></li> <!--Mobile calendar/contact integration-->
 							<li title="With Collabora Online" rel="tooltip"><i class="fa fa-usd" aria-hidden="true"></i></li> <!--Online Office-->
-							<li title="" rel="tooltip"><i class="fa fa-check" aria-hidden="true"></i></li> <!--File Drop (customer file upload)-->
+							<li title="Only in ownCloud 10" rel="tooltip"><i class="fa fa-check" aria-hidden="true"></i></li> <!--File Drop (customer file upload)-->
```
Comment (MariusBluem) [inline page-compare.php]: This is nothing I would mention (only in v10)

File: page-compare.php (hunk 1 above)
Comment (jospoortvliet) [inline page-compare.php]: the app store is beta...

File: page-compare.php (hunk 2 above)
Comment (jospoortvliet) [inline page-compare.php]: mja... I thought it's useful info - this isn't in what most customers would deploy - many know 10 isn't really ready yet, just look at the beta app store.

**Pull request merged**